backend/src/lang/intents/factory/factory.enddateopen.ts

```diff
@@ -0,0 +1,29 @@
+import { IntentHandler } from './../handlers/handler.abstract';
+import { EndDateOpenIntentHandler } from '../handlers/handler.enddateopen';
+import { IIntentFactory } from './factory.interface';
+import { Injectable } from '@nestjs/common';
+import IntentConfig from './../IntentConfig';
+
+@Injectable()
+export class EndDateOpenFactory implements IIntentFactory {
```

Comment:
Warum zwei Handler/Intents für EndDate? Wäre es nicht schöner alles in einem zu machen?